mautrix/util/proxy.py

```diff
@@ -102,7 +103,9 @@ async def proxy_with_retry(
             errors += 1
             if errors > max_retries:
                 raise
-            wait = min(errors * 10, min_wait_seconds)
+            wait = errors * 10
```

Comment:
Should probably add a parameter for the multiplier too just to be safe

Reply:
https://github.com/mautrix/python/pull/137/commits/b890d290f85478b2062854584d6f79e0dd84efd0